lightning/src/ln/channelmanager.rs

```diff
@@ -3379,16 +3380,93 @@ where
 			*best_block = BestBlock::new(header.prev_blockhash, new_height)
 		}
 
-		self.do_chain_event(Some(new_height), |channel| channel.update_best_block(new_height, header.time));
+		self.do_chain_event(Some(new_height), |channel| channel.best_block_updated(new_height, header.time));
+	}
+}
+
+impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> chain::Confirm for ChannelManager<Signer, M, T, K, F, L>
+where
+	M::Target: chain::Watch<Signer>,
+	T::Target: BroadcasterInterface,
+	K::Target: KeysInterface<Signer = Signer>,
+	F::Target: FeeEstimator,
+	L::Target: Logger,
+{
+	/// Updates channel state to take note of transactions which were confirmed in the given block
```

Comment:
If we drop the docs completely rustdoc will just pick up the docs from the trait itself, which I think is what we want.